tests/core/badge_awarded/user_achieved_badge/statements.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "actor": {
+      "name": "test_recipient_firstname test_recipient_lastname",
+      "account": {
+        "homePage": "http://www.example.org",
+        "name": "1"
+      }
+    },
+    "verb": {
+      "id": "https://w3id.org/xapi/tla/verbs/achieved",
+      "display": {
+        "en": "Achieved"
+      }
+    },
+    "object": {
+      "id": "http://www.example.org/badges/overview.php?id=1",
+      "objectType": "Activity",
+      "definition": {
+          "name": {"en":"test_badgename"},
+        "description": {"en": "test badge description"},
+        "type": "https://xapi.edlm/profiles/edlm-lms/concepts/activity-types/badge",
+        "extensions": {
+          "https://xapi.edlm/profiles/edlm-lms/v1/concepts/activity-extensions/badge-type": "Course",
+          "https://xapi.edlm/profiles/edlm-lms/v1/concepts/activity-extensions/badge-version": "1.0"
+        }
+      }
+    },
+    "context": {
+      "language": "en",
+      "instructor": {
+        "name": "test_awarder_firstname test_awarder_lastname",
+        "account": {
+            "homePage": "http://www.example.org",
+            "name": "2"
+        }
+      },
+      "contextActivities": {
+        "category": {
+          "id": "http://www.example.org",
+          "objectType": "Activity",
+          "definition": {
+            "name": {
+              "en": "EDLM Moodle LMS"
+            },
+            "type": "http://id.tincanapi.com/activitytype/lms"
+          }
+        },
+        "parent": {
+          "id": "http://www.example.org/course/view.php?id=1",
+          "objectType": "Activity",
+          "definition": {
+            "name": {
+              "en": "test_course_name"
+            },
+            "description": {
+              "en": "test_course_summary"
+            },
+            "type": "https://w3id.org/xapi/cmi5/activitytype/course"
+          }
+        }
+      },
+      "extensions": {
+        "https://xapi.edlm/profiles/edlm-lms/v1/concepts/context-extensions/badge-assignment-method": "Manual",
+        "http://lrs.learninglocker.net/define/extensions/info": {
```

Comment:
I think we're gonna change this key but I will do it globally in another PR so this one is fine for now